src/iris-mobile/.gitignore

```diff
@@ -0,0 +1,2 @@
+keys.py
+*.pyc
```

Comment:
Thanks, for the input. I need to remove the whole src/iris-mobile directory since it's integrated into into the main iris api already and I can see I missed moving some things to config files there. Will be working on fixing the flake8 issues right now.